chringo/lib/main.dart

```diff
@@ -1,89 +1,14 @@
-import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:chringo/pages/bingo_page.dart';
```

Comment:
Package organization/library-tization is going to play a huge part in keeping our code organized and modular. More to come on that in a bit. For now, let's standardize our import order. I suggest:
```dart
// dart-level libraries first
import 'dart:math';

// then external libraries
import 'package:flutter/material.dart';

// then our own libs
import 'package:chringo/pages.dart';
```
with a space between each.

Reply:
Should our libs have `package:` as a prefix? It works both ways - what’s that difference?

Reply:
We should probably use relative imports for our own code, but honestly we should import libraries that we need in other places. We want low-cohesion, so putting things in their own logical package or context is important